test/promql/tests/data_plane/eaas_test.yaml

```diff
@@ -0,0 +1,26 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.eaas_alerts.yaml
+
+tests:
+  - interval: 1m
+    input_series:
+      - series: platform:hypershift_hostedclusters:max{platform="AWS"}
```

Comment:
The rule is using `sum`. I think we should test it with multiple time series.

Reply:
Is that quota global to all our clusters?

Reply:
Good point. I split the time series in two.